packages/client/src/Dashboard.tsx

```diff
@@ -165,17 +175,19 @@ export default function Dashboard({ accountId, navigate }: Props) {
         />
       ) : isSearching ? (
         <SearchBar
-          onChange={setSearchQuery}
+          onChange={(q: string) =>
+            setQuery({ searchQuery: q, isSearching: true }, "replace")
+          }
           onClose={() => {
-            setIsSearching(false)
+            setQuery({ searchQuery: "", isSearching: false }, "replace")
```

Comment:
I would rather not unset the search query when the search input is closed. If the user closes search, and opens it again I want the query and search results that they previously entered to still be there.